<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Webtoon Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_search">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_source">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string notr="true">Naver</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Naver Best Challenge</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_search">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>0</y>
        <width>601</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string notr="true">Search</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>0</y>
        <width>47</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>798</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea_results">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>801</width>
        <height>389</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_results">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>799</width>
         <height>387</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>801</width>
          <height>391</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="topMargin">
          <number>5</number>
         </property>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_downloads">
     <attribute name="title">
      <string>Downloads</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea_downloads">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>451</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::VLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>799</width>
         <height>449</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionShow_Log">
   <property name="text">
    <string>Show Log</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
